tests/testSN.py

```diff
@@ -328,39 +340,39 @@ class MyGalaxyCatalog(CatalogDBObject):
             raColName = 'raJ2000'
             decColName = 'decJ2000'
             driver = 'sqlite'
-        
+
             # columns required to convert the ra, dec values in degrees
             # to radians again
             columns = [('id', 'id', int),
-                       ('raJ2000','raJ2000 * PI()/ 180. '),
-                       ('decJ2000','decJ2000 * PI()/ 180.'),
+                       ('raJ2000', 'raJ2000 * PI()/ 180. '),
+                       ('decJ2000', 'decJ2000 * PI()/ 180.'),
                        ('redshift', 'redshift')]
 
-	# class galCopy(InstanceCatalog):
```

Comment:
I'm not sure.  Rahul would know best.  I erred on the sign of not changing anything I didn't have to when I ported this over.